### ISSUE TITLE
Linter Proposal: LeadingCommentSpace

### ISSUE THREAD

**Issue by ivantsepp** [brigade/scss-lint#813]
I would like to propose a linter that's similar to what [Rubocop](http://www.rubydoc.info/github/bbatsov/rubocop/Rubocop/Cop/Style/LeadingCommentSpace) has. It would throw a warning if there is no leading space in comments.

```scss
//bad comment
/*bad comment */
```

```scss
// good comment
/* good comment */
```

Let me know what you think! I would like to also implement this linter if you agree with this =)

**Comment by sds** [comment]
Implemented in #825
